runtime/docker/docker.go

```diff
@@ -234,8 +234,11 @@ func (d *DockerRuntime) postCreateNetActions() (err error) {
 	if err != nil {
 		log.Warnf("failed to disable TX checksum offloading for the %s bridge interface: %v", d.mgmt.Bridge, err)
 	}
-
-	return d.installIPTablesFwdRule()
+	err = d.installIPTablesFwdRule()
```

Comment:
I was thinking that maybe we should change the `install/deleteIPTablesFwdRule` funcs to return both an error and the stderr.
It seems that with the current way we invoke os/exec the error message from iptables is not captures, since stderr is not captured

https://github.com/LimeHat/containerlab/blob/121059d710bbf00d2616e9cad13d0d2d63f5bb29/runtime/docker/iptables.go#L31

For example, I believe that if we would change the command to this:

```
❯ iptables -I FAKECHAIN -o eth0 -j ACCEPT
iptables: No chain/target/match by that name.
```
we wouldn't display the error message, and just report "exited with status 1", which is fine, but doesn't give a hint about the probable cause.

So maybe what makes sense is to use `.CombinedOutput` for the `os.exec/Cmd` call, this will capture the stderr and stdout (ofc we can just capture the stderr, but more code lines required :D)
Then should the error be non nil, we create an error with the combined output provided for the context.


PS. (same will be needed for delete command as well)